lib/CryptoLD.js

```diff
@@ -67,10 +67,63 @@ class CryptoLD {
     return this.suites.get(type).from(serialized);
   }
 
+  /**
+   * Imports a key pair instance via the provided `documentLoader` function,
+   * optionally checking it for revocation and required context.
+   *
+   * @param {string} keyId - Key ID or URI.
+   * @param {Function} documentLoader - JSON-LD Document Loader.
+   * @param {boolean} [checkContext=true] - Whether to check that the fetched
+   *   key document contains the context required by the key's crypto suite.
+   * @param {boolean} [checkRevoked=true] - Whether to check the key object for
+   *   the presence of the `revoked` timestamp.
+   *
+   * @returns {Promise<LDKeyPair>} Resolves with the appropriate key pair
+   *   instance.
+   */
+  async fromKeyId({
+    keyId, documentLoader, checkContext = true, checkRevoked = true
+  } = {}) {
+    if(!keyId) {
+      throw new TypeError('The "keyId" parameter is required.');
+    }
+    if(!documentLoader) {
+      throw new TypeError('The "documentLoader" parameter is required.');
+    }
+    let keyDocument;
+    try {
+      ({doc: keyDocument} = await documentLoader(keyId));
+    } catch(e) {
+      const error = new Error('Error fetching document: ' + e.message);
+      error.cause = e;
+      throw error;
+    }
+    const fetchedType = keyDocument.type;
+    if(!fetchedType) {
+      throw new Error('Key suite type not found in fetched document.');
+    }
+    const keySuite = this.suites.get(fetchedType);
+    if(!keySuite) {
+      throw new Error(`Support for key type "${keySuite}" is not installed.`);
+    }
+    if(checkContext) {
+      const fetchedDocContexts = [].concat(keyDocument['@context']);
+      if(!fetchedDocContexts.includes(keySuite.SUITE_CONTEXT)) {
+        throw new Error('Fetched document does not contain required context "' +
+          keySuite.SUITE_CONTEXT + '".');
+      }
+    }
+    if(checkRevoked && keyDocument.revoked) {
+      // eslint-disable-next-line max-len
```

Comment:
can we remove this eslint max length rule?

Reply:
Sure, rephrased the error to remove.